lib/ws_clients/hf_ds/on_backtest_candle.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+const send = require('../../util/ws/send')
+
+/*
+
+    ['data.sync.start', exchange, symbol, timeFrame, from, to, meta] - when a sync process starts
+    ['data.sync.end', exchange, symbol, timeFrame, from, to, meta] - when a sync process end
+    ['bt.start', null, null, from, to, null, nTrades, nCandles] - before the backtest data stream
+    ['bt.end', null, null, from, to] - after the backtest data stream
+    ['bt.candle', null, null, candle] - individual BT candle
+    ['bt.trade', null, trade] - individual BT trade
+
+*/
+
+// NOTE: Data server symbol may differ from the requested symbol
+module.exports = (dsClient, msg) => {
```

Comment:
Can you find a way to generate these handlers, since they all have a similar form, with a payload transformer func or so? i.e. a `createDSProxyHandler('bt.candle', p => p)` where 2nd arg is transformer (could be done so it is unnecessary if all data is kept as-is when proxying), you can just strip the `reqID` off the end.

Reply:
Done, good idea